app/managers.py

```diff
@@ -0,0 +1,37 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self):
+        self._connection = sqlite3.connect("cinema.db3")
+        self.table_name = "cinemas"
+
+    def create(self, first_name: str, last_name: str):
+        self._connection.execute(
+            f"INSERT INTO {self.table_name} (first_name, last_name)"
+            f"VALUES (?, ?)", (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self):
+        cinemas_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [Actor(*row) for row in cinemas_cursor]
+
+    def update(self, new_id: int, new_first_name: str, new_last_name: str):

```

Comment:
you call the parameter `new_id`, although in fact id is old and it does not change